src/App.asm

```diff
@@ -1,6 +1,137 @@
 .data
+str1: .asciii "Enter total number of integers: "
 
+str2: .asciii "Enter an integer: "
+
+str3: .asciii "Total pos_count: "
+
+str4: .asciii "Total neg_count: "
+
+new_line: .asciii "\n"
+
+value: .word 0
+ 
+i: .word 0 
+
+pos_count: .word 0
+
+neg_count: .word 0
+
+input: .word 0
 
 .globl main
 
-.text
\ No newline at end of file
+.text
+
+main:
+# display str1
+
+li	$v0, 4
+la	$a0, str1
+syscall
+
+# read value
+
+li	$v0, 5
+syscall
+
+move  $t1, $v0
+ 
+sw $t1, value
+lw $t2, i
+lw $t1, value
+lw $t5, neg_count
+
+
+ Begain_loop:
```

Comment:
mips is case sensitive, here you defined the label as `Begin_loop` with big `B` but the reset code you use it as `begain_loop` which small `b`.